packages/nuxt/README.md

```diff
@@ -0,0 +1,77 @@
+# Vant Nuxt
+
+> Vant module for Nuxt
+
+## Features
+
+- Automatically import components and styles on demand.
+- Automatically import of showDialog, showToast and other methods.
+
+## Install
+
+```shell
+npm i @vant/nuxt -D
+```
+
+```ts
+export default defineNuxtConfig({
+  modules: ['@vant/nuxt'],
+  vant: {
+    /** Options */
+  },
+});
+```
+
+## Usage
+
+```vue
+<template>
+  <van-button type="primary" @click="showToast('toast')"> button </van-button>
+  <VanButton type="success" @click="showNotify('notify')"> button </VanButton>
+  <LazyVanButton type="default"> lazy button </LazyVanButton>
```

Comment:
What is the difference between VanButton and LazyVanButton

Reply:
[dynamic-imports](https://nuxt.com/docs/guide/directory-structure/components#dynamic-imports)